eq_translations/translatable_item.py

```diff
@@ -1,7 +1,7 @@
 from dataclasses import dataclass
 from typing import Union, Optional, List
 
-
+# pylint: disable=unsubscriptable-object
```

Comment:
Is this actually needed?. I removed it and the lint passed file with 3.9 with latest lock file.

Reply:
Just reverted and the build is falling over again, so not sure why yours is passing and mine is not 😐

Reply:
Failed in my local 3.9 build too.

Reply:
Ya my mistake, can reproduce it now. Open issue: https://github.com/PyCQA/pylint/issues/3882